portfolio.css

```diff
@@ -0,0 +1,102 @@
+* {
+	margin: 0 auto;
+	font-size: 20px;
+	color: #ff4716;	
+	padding: 1px;
+	font-family: 'Comfortaa', sans-serif, cursive;
+}
+
+.navbar a {
+	padding: 50% 0 0 5%;
+	text-decoration: none;
+	margin: 50% 0 50% 0;
+	position: relative;
+	top: 30px;
+	font-size: 1.5rem;	
+}
+
+.navbar {
+	background-color: #feeeeb;
+	padding: 0 0 30px;
+}
+
+.welcome-banner {
```

Comment:
This inherits a 1px padding from * { }. For me it makes it look odd and out of line with other parts of the page.

Reply:
Padding removed